### ISSUE TITLE
New routes resulting in 404

### ISSUE THREAD

**Issue by paragonsoftwaresolutions** [tuandm/laravue#64]
I am trying to implement a "Create" and "Edit" button that will open the desired page. I set the routes up the same as it is done in the admin routes file for articles/create and articles/edit. However, when I click on the button I get a 404 response. I am not seeing a route set up for articles/create yet it still works. Am I missing something here?

Here is my router module:
`import Layout from '@/layout';

const productsRoutes = {
  path: '/products',
  component: Layout,
  redirect: '/products/catalog',
  name: 'Products',
  alwaysShow: true,
  meta: {
    title: 'Products',
    icon: 'shopping',
    permissions: ['view menu products'],
  },
  children: [
    {
      path: 'catalog',
      component: () => import('@/views/products/List'),
      name: 'Catalog',
      meta: { title: 'Catalog', icon: 'list', noCache: true },
    },
    {
      path: 'products/create',
      component: () => import('@/views/products/Create'),
      name: 'CreateProduct',
      meta: { title: 'createProduct', icon: 'edit', permissions: ['manage products'] },
      hidden: true,
    },
    {
      path: 'products/edit/:id(\\d+)',
      component: () => import('@/views/products/Edit'),
      name: 'EditProduct',
      meta: { title: 'editProduct', noCache: true, permissions: ['manage products'] },
      hidden: true,
    },
  ],
};

export default productsRoutes;`

Here is the code for the button to open the page:
`<router-link :to="'/products/edit/'+row.id">
                        <el-button type="primary" size="small" icon="el-icon-edit" v-permission="['manage products']">
                            Edit
                        </el-button>
                    </router-link>`

**Comment by tuandm** [comment]
@paragonsoftwaresolutions Have you added `productsRoutes` to your `router/index.js`? There are some examples in Laravue that you can use as references - like user, article.

**Comment by paragonsoftwaresolutions** [comment]
Yes, it is there. I can access Catalog page, but I can not access the create or edit page.

**Comment by tuandm** [comment]
@paragonsoftwaresolutions 
```
path: 'products/edit/:id(\d+)',
```
I think this should be `\\d` because you are using single quote.

**Comment by paragonsoftwaresolutions** [comment]
I have tried it that way. Same results. I can't access the create page either.

**Comment by tuandm** [comment]
@paragonsoftwaresolutions Thanks for letting me know. I'm sorry because I couldn't help you.
